assets/src/edit-story/components/inspector/prepublish/styles.js

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import {
+  Headline,
+  Icons,
+  Text,
+  themeHelpers,
+  THEME_CONSTANTS,
+} from '../../../../design-system';
+import { MAX_NUMBER_FOR_BADGE } from './constants';
+
+function annotateNumber(number) {
+  if (number <= MAX_NUMBER_FOR_BADGE) {
+    return number;
+  }
+  return `${MAX_NUMBER_FOR_BADGE}+`;
+}
+
+export const NumberBadge = styled.span`
+  display: inline-flex;
+  height: 20px;
+  width: 20px;
+  line-height: 20px;
+  justify-content: center;
+  border-radius: 50%;
+  font-size: ${({ number }) =>
+    number > MAX_NUMBER_FOR_BADGE ? '10px' : '12px'};
+  &::after {
+    content: ${({ number }) => `"${annotateNumber(number)}"`};
+  }
+  color: ${({ theme }) => theme.colors.bg.secondary};
+  background-color: ${({ isRecommended, theme }) =>
+    isRecommended ? theme.colors.fg.linkNormal : theme.colors.fg.negative};
+`;
+NumberBadge.propTypes = {
+  isRecommended: PropTypes.bool,
+  number: PropTypes.number,
+};
+
+export const TitleWrapper = styled.span`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
```

Comment:
this isn't grabbing enough width yet in the title area but this is as high up as we can go with just the prepublish styling.